std/math.d

```diff
@@ -7444,6 +7444,34 @@ private real polyImpl(real x, in real[] A) @trusted pure nothrow @nogc
                 ;
             }
         }
+        else version (DragonFlyBSD)
+        {
+            asm pure nothrow @nogc // assembler by W. Bright
```

Comment:
For the record: I'm not a huge fan of copying blocks as this means a bug fix will very likely miss one of these bits.
Anyhow as far as I know is Walter a huge proponent of using `version(A)` and not of `static if( A || B)`

Reply:
I completely agree. I did see this 'argument' pop up several time in different PR's and Forum discussions. I even wanted to suggest a generic 'BSD' version flag, as they all have common origins, similar to the 'Posix' version. But i did not want to reopen this discussion.

Reply:
Or maybe we do away with the asm implementation and implement a better poly algorithm.

https://issues.dlang.org/show_bug.cgi?id=12084

Reply:
It's possible too to put the implementation in a token string  `enum polyAsm = q{ asm pure nothrow @nogc { ... } };` and to mix it

Reply:
I think this discussion should be had outside this PR. Sounds more like a structural (ie ASM) discussion, that pertaining to this particular PR (correct).